CORRELATION_RULES_STYLE_GUIDE.md

```diff
@@ -0,0 +1,70 @@
+# panther-analysis Correlation Rule (CR) Style Guide and Best Practices
+
+This style guide highlights essential best practices for writing correlation rules. For a more detailed guide, visit [Correlation Rules](https://docs.panther.com/detections/correlation-rules) in the Panther documentation.
```

Comment:
I loved that you added the acronym "(CR)," but we typically don't add those in titles. so like the title would just say Correlation Rule, and the first use of it (in this sentence) would have (CR) after